scripts/config.py

```diff
@@ -89,9 +89,10 @@
     AUTH_IS_LOGGED_IN_INDICATOR = config["scan"]["scriptAuth"]["authIsLoggedInIndicator"]
     AUTH_INDICATOR_REGEX = config["scan"]["scriptAuth"]["authIndicatorRegex"]
 
-    # HTTP Sender script
-    USE_HTTP_SENDER_SCRIPT = config["scan"]["scriptAuth"]["useHttpSenderScript"]
-    HTTP_SENDER_SCRIPT_NAME = config["scan"]["scriptAuth"]["HttpSenderScriptName"]
-    HTTP_SENDER_SCRIPT_ENGINE = config["scan"]["scriptAuth"]["HttpSenderScriptEngine"]
-    HTTP_SENDER_SCRIPT_FILE_PATH = config["scan"]["scriptAuth"]["HttpSenderScriptFilePath"]
-    HTTP_SENDER_SCRIPT_DESCRIPTION = config["scan"]["scriptAuth"]["HttpSenderScriptDescription"]
+
```

Comment:
I would add a `if getattr(config["scan"], "useHttpSenderScript", false):` to avoid crashing when the user does not configure an httpsender script